Libraries/ConfigParser/inc/config_parser.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "structs.h"
+
+namespace StreamingToolkit
+{
+	class ConfigParser
```

Comment:
this being a wrapper for static methods will hurt our ability to test - it would be great it we could leverage a factory pattern or some DI to get an instance of a thing capable of parsing, then in tests we could instead return a mocked parser that returns values we want, without needing to hit the file system

Reply:
Would you like to help with this improvement? Feel free to go ahead and push your commit. :)

Reply:
sure, i'll see if i can make some quick adjustments here ASAP